lineage/states/stateCommon.py

```diff
@@ -68,15 +68,14 @@ def gamma_estimator(gamma_obs, time_cen, gammas, x0):
     assert gammas.shape[0] == gamma_obs.shape[0]
     arrgs = (gamma_obs[time_cen == 1], gammas[time_cen == 1], gamma_obs[time_cen == 0], gammas[time_cen == 0])
     opt = {'gtol': 1e-12, 'ftol': 1e-12}
-    bnd = (1.0, 800.0)
+    bnd = (1.0, 100.0)
```

Comment:
I assume this bound is still sufficiently wide to cover all feasible values?

Reply:
Oh I tried this to fix the error in the tests. I was passing almost 99% when I changed it to 100. I'm not sure if this was the problem though. Should I bring that back?

Reply:
To answer your question, I think it is.

Reply:
Ok, this is good to merge.